localdev/config/lookout/config.yaml

```diff
@@ -13,10 +13,10 @@ grpc:
     permitWithoutStream: false
 
 uiConfig:
-  armadaApiBaseUrl: "http://armada-server:8080"
+  armadaApiBaseUrl: "http://localhost:8080"
```

Comment:
Why is this being done? Shouldn't we default to the docker network namespace?

Reply:
this url is used in web ui (browser) so the internal/docker name doesn't work

Reply:
Ah yeah good point, because those names won't resolve on a MacOS Host.